expression/builtin_miscellaneous.go

```diff
@@ -1043,5 +1044,36 @@ type uuidShortFunctionClass struct {
 }
 
 func (c *uuidShortFunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (builtinFunc, error) {
-	return nil, errFunctionNotExists.GenWithStackByArgs("FUNCTION", "UUID_SHORT")
+	if err := c.verifyArgs(args); err != nil {
+		return nil, err
+	}
+	bf, err := newBaseBuiltinFuncWithTp(ctx, c.funcName, args, types.ETInt)
+	if err != nil {
+		return nil, err
+	}
+
+	sig := &builtinUUIDShortSig{bf}
+	return sig, nil
+}
+
+type builtinUUIDShortSig struct {
+	baseBuiltinFunc
+}
+
+func (b *builtinUUIDShortSig) Clone() builtinFunc {
+	newSig := &builtinUUIDShortSig{}
+	newSig.cloneFrom(&b.baseBuiltinFunc)
+	return newSig
+}
+
+// evalInt evals a builtinUUIDShortSig.
+// See https://dev.mysql.com/doc/refman/8.0/en/miscellaneous-functions.html#function_uuid-short
+func (b *builtinUUIDShortSig) evalInt(_ chunk.Row) (d int64, isNull bool, err error) {
+	now, seq, err := uuid.GetTime()
+	if err != nil {
+		return d, false, err
+	}
+	serverID := b.ctx.GetSessionVars().ConnectionInfo.ServerID
```

Comment:
I was looking to make sure this is accurate. I'm not sure it is because of https://github.com/pingcap/tidb/blob/master/server/server.go#L490

But it is an easy fix, since varying serverID can be fetched with:
https://github.com/pingcap/tidb/blob/master/server/server.go#L664-L667